Ktisis/Interface/Components/Chara/CharaImportUI.cs

```diff
@@ -35,7 +37,9 @@ FileSelect<CharaFile> select
 	// Initialization
 
 	public void Initialize() {
```

Comment:
is there any performance downsides to refactoring this to remove the two-stage init?

I personally find this quite an antipattern, and it's possible that one-time init might be enough for this, instead of going by the each UI load